requirements.txt

```diff
@@ -1,2 +1,16 @@
-Werkzeug==0.11.2
```

Comment:
I generated this in a clean dev environment, and this should be used for development and setup.py's install_requires should be the canonical dependency list.